README.md

```diff
@@ -118,6 +119,49 @@ goldrush run reads=long_reads G=3e9
 
 **For more information about the GoldRush algorithm and tips for running GoldRush see our [wiki](https://github.com/bcgsc/goldrush/wiki)** 
 
+# System Requirements
+
+## Hardware Requirements
+
+GoldRush requires a standard computer with enough RAM to support the operations defined by the user. For the assembly of a human genome with a ~60X coverage, GoldRush requires a computer with:
+
+RAM: 64+ GB 
+
+and recommends a computer with at least:
+
+CPU: 24+ cores (48+ threads).
+
+The runtimes and RAM usage below are generated using a server-class system (144 Intel(R) Xeon(R) Gold 6254 CPU @ 3.1 GHz with 2.9 TB RAM) with 48 threads specified.
+
+| Data        | Time (h)    | RAM (GB)    |
+| ----------- | ----------- | ----------- |
+| 67X *H. sapiens*  | 16.6       | 51.9        |
+| 63X *H. sapiens*  | 20.8       | 53.9        |
+| 71X *H. sapiens*  | 20.8       | 54.5        |
+
+## Software Requirements
+
+### OS Requirements
+
+All package development version is tested on *Linux* operating systems. The developmental version of the package has been tested on the following systems:

```

Comment:
```suggestion
GoldRush has been tested on *Linux* operating systems (centOS7)

```
Just suggestion to make this more concise, and wasn't too sure what you meant by "developmental version"

Reply:
its phrased that way because of "versions the software has been tested on" from the checklist

Reply:
If we're saying that, we have also tested on Ubuntu (via CI)? And would listing the versions in parenthesis not still answer that point?